crates/bevy_pbr/src/render/skinning.wgsl

```diff
@@ -22,10 +22,10 @@ fn skin_model(
     indexes: vec4<u32>,
     weights: vec4<f32>,
 ) -> mat4x4<f32> {
-    var matrix = weights.x * joint_matrices.data[indexes.x];
-    matrix = add_matrix(matrix, weights.y * joint_matrices.data[indexes.y]);
-    matrix = add_matrix(matrix, weights.z * joint_matrices.data[indexes.z]);
-    return add_matrix(matrix, weights.w * joint_matrices.data[indexes.w]);
+    var skinned_matrix = weights.x * joint_matrices.data[indexes.x];
+    skinned_matrix = add_matrix(skinned_matrix, weights.y * joint_matrices.data[indexes.y]);
```

Comment:
I think add_matrix may no longer be needed, as the + operator works on mat4x4 in wgpu 0.13 as long as you aren't on an old version of naga I guess.

Reply:
indeed https://github.com/gfx-rs/naga/issues/1527 should be OK now, but no way to test for me 👍